packages/contracts/contracts/optimistic-ethereum/OVM/chain/OVM_CanonicalTransactionChain.sol

```diff
@@ -971,103 +973,146 @@ contract OVM_CanonicalTransactionChain is iOVM_CanonicalTransactionChain, Lib_Ad
     }
 
     /**
-     * Checks that a given batch context has a time context which is below a given que element
-     * @param _context The batch context to validate has values lower.
-     * @param _queueIndex Index of the queue element we are validating came later than the context.
+     * Checks that a pair of adjacent batch contexts are monotonic with respect to each other,
+     * and with respect to any queue elements which may be appended between them.
```

Comment:
Can we add a line here that it `Also checks that any queue elements between are not yet older than the force inclusion period, which would force them to come next.`?